lib/friendly_time_diff.rb

```diff
@@ -29,7 +29,7 @@ def date_parts
     end
 
     def date_part(label, amount)
-      amount > 0 ? pluralize(amount, label) : nil
+      amount > 0 ? [amount, label.pluralize(amount)].join(" ") : nil
```

Comment:
I think something like `"#{amount} #{label.pluralize(amount)}"` would be easier to read here. What do you think?